test-suite/mk-tls-test-certs.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+#
+# test certificates and keys
+#
+
+if test -x /usr/bin/certtool; then
+
+echo "
+organization = \"Example.Org\"
+cn = \"Test CA. Do not Trust\"
+dc = \"example.org\"
+expiration_days = 700
+ca
+cert_signing_key
+crl_signing_key
+tls_www_server
+" >example.org-ca
```

Comment:
Here and elsewhere in this PR, please add a customary space between `>` and `x` in ` >x` expressions and alike.

Reply:
done